docs/day-2-operations/Backups.md

```diff
@@ -0,0 +1,272 @@
+   ```
+
+### `system-redis`
+
+1. Backup the dump.rb file from redis
+
+   ```
+   oc cp $(oc get pods -l 'deploymentConfig=system-redis' -o json | jq '.items[0].metadata.name' -r):/var/lib/redis/data/dump.rdb ./system-redis-dump.rdb
+   ```
+
+## Restore Procedures
+
+### `system-mysql`
+
+1. Copy the MySQL dump to the system-mysql pod
+   ```
+   oc cp ./system-mysql-backup.gz $(oc get pods -l 'deploymentConfig=system-mysql' -o json | jq '.items[0].metadata.name' -r):/var/lib/mysql
+   ```
+2. Decompress the Backup File
+    ```
+    oc rsh $(oc get pods -l 'deploymentConfig=system-mysql' -o json | jq -r '.items[0].metadata.name') bash -c 'gzip -d ${HOME}/system-mysql-backup.gz'
+    ```
+3. Restore the MySQL DB Backup file
+   ```
+   oc rsh $(oc get pods -l 'deploymentConfig=system-mysql' -o json | jq -r '.items[0].metadata.name') bash -c 'export MYSQL_PWD=${MYSQL_ROOT_PASSWORD}; mysql -R -hsystem-mysql -uroot system < ${HOME}/system-mysql-backup'
+   ```
+
+### `system-storage`
+
+1. Restore the Backup file to system-storage
+   ```
+   oc rsync ./local/dir $(oc get pods -l 'deploymentConfig=system-app' -o json | jq '.items[0].metadata.name' -r):/opt/system/public/system
+   ```
+
+### `zync-database`
+
+1. Copy the Zync Database dump to the zync-database pod
+   ```
+   oc cp ./zync-database-backup.gz $(oc get pods -l 'deploymentConfig=zync-database' -o json | jq '.items[0].metadata.name' -r):/var/lib/pgsql/
+   ```
+2. Decompress the Backup File
+    ```
+    oc rsh $(oc get pods -l 'deploymentConfig=zync-database' -o json | jq -r '.items[0].metadata.name') bash -c 'gzip -d ${HOME}/zync-database-backup.gz'
+    ```
+3. Restore the PostgreSQL DB Backup file
+   ```
+   oc rsh $(oc get pods -l 'deploymentConfig=zync-database' -o json | jq -r '.items[0].metadata.name') bash -c 'psql -f ${HOME}/zync-database-backup'
+   ```
+### `backend-redis`
+* After restoring `backend-redis` a sync of the Config information from `System` should be forced, to ensure the information in `Backend` is consistent with that in `System` (the source of truth).
+
+1. Edit the `redis-config` configmap
+   ```
+   oc edit configmap redis-config
+   ```
+
+2. Comment `SAVE` comands in the `redis-config` configmap
+   ```
+    #save 900 1
+    #save 300 10
+    #save 60 10000
+   ```
+
+3. Set `appendonly` to no in the `redis-config` configmap
+   ```
+   appendonly no
+   ```
+
+4. Re-deploy `backend-redis` to load the new configurations
+   ```
+   oc rollout latest dc/backend-redis
+   ```
+
+5. Rename the `dump.rb` file
+   ```
+   oc rsh $(oc get pods -l 'deploymentConfig=backend-redis' -o json | jq '.items[0].metadata.name' -r) bash -c 'mv ${HOME}/data/dump.rdb ${HOME}/data/dump.rdb-old'
+   ```
+
+6. Rename the `appendonly.aof` file
+   ```
+   oc rsh $(oc get pods -l 'deploymentConfig=system-redis' -o json | jq '.items[0].metadata.name' -r) bash -c 'mv ${HOME}/data/appendonly.aof ${HOME}/data/appendonly.aof-old'
```

Comment:
the command references 'system-redis'

Reply:
fixed on 580a5f5dca8d02c8d085be2c5ab700ff53cb7e89